errbot/repo_manager.py

```diff
@@ -154,7 +154,13 @@ def index_update(self) -> None:
         for source in reversed(self.plugin_indexes):
             try:
                 if urlparse(source).scheme in ('http', 'https'):
-                    with urlopen(url=source, timeout=10) as request:  # nosec
+                    with urlopen(
+                        Request(
+                            source,
+                            headers={'User-Agent': 'Errbot'}
+                        ),
+                        timeout=10
+                    ) as request:  # nosec
```

Comment:
I think it is a bit more legible breaking it up into multiple lines.

```suggestion
                    req = Request(source, headers={'User-Agent': 'Errbot'})
                    with urlopen(url=req, timeout=10) as request:  # nosec
```